shared/src/commonMain/kotlin/com/learndiscussvote/projectguardian/shared/PoliticalRaceModel.kt

```diff
@@ -48,11 +54,14 @@ enum class PoliticalStatementCategory {
     HEALTH,
     ABORTION,
     RIGHTS_EQUALITY,
-    TAXES,
-    GOVERNMENT,
+    TAXES_FUNDING
+    GOVERNMENT OPERATION,
```

Comment:
```suggestion
    GOVERNMENT_OPERATION,
```

Can't have spaces unfortunately